ads/_config.js

```diff
@@ -105,6 +105,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  adocean: {},
```

Comment:
could you implement renderStart API? check out the [guidelines](https://github.com/ampproject/amphtml/blob/master/ads/README.md#developer-guidelines-for-a-pull-request)